main.go

```diff
@@ -512,7 +512,10 @@ func _main(ctx context.Context) error {
 	var _ agendas.DeploymentSource = dcrdClient
```

Comment:
Please remove this line while here.  Passing dcrdClient to CheckAgendasUpdates accomplishes the same complile time check.